config/routes.rb

```diff
@@ -85,6 +84,7 @@
   post 'phylo_trees/retry', to: 'phylo_trees#retry'
   get 'phylo_trees/:id/download', to: 'phylo_trees#download'
   get 'choose_taxon', to: 'phylo_trees#choose_taxon'
+  get 'choose_taxon_all_levels', to: 'samples#choose_taxon_all_levels'
```

Comment:
As said before, from an API perspective, makes more sense a route to get taxons, and then add a param that can constrain to certain levels, no?

Reply:
ok